src/Entities/Donation.php

```diff
@@ -777,21 +786,7 @@ public function modifyDataObject( callable $modificationFunction ) {
 	 *
 	 * @return AddressChange
 	 */
-	public function getAddressChange(): ?AddressChange {
+	public function getAddressChange(): AddressChange {
 		return $this->addressChange;
 	}
-
-	/**
-	 * Set AddressChange reference
-	 *
-	 * @since 8.0
```

Comment:
Oh no :D Now it needs to be 9.0. Unless you want to pretend it never happened :) Which you can do since the fun team is the only user of this component anyway.